internal/renderers/skia/lib.rs

```diff
@@ -68,7 +68,13 @@ pub struct SkiaRenderer {
 }
 
 impl SkiaRenderer {
-    /// Creates a new renderer is associated with the provided window adapter.
+    /// Creates a new renderer for the given window/display handle. The provided size
+    /// must match the size of the underlying native window.
+    /// 
+    /// Subsequent calls to [`Self::render()`] will render into the window.
+    /// 
+    /// Safety: It is the caller's responsibility to ensure that the underlying native
+    /// window exists and remains valid for the life-time of the SkiaRenderer.
```

Comment:
I wonder if the constructor function here should be marked unsafe because of that.

Reply:
That depends what happens if the underlying native window becomes invalid. If we detect it and assert, it's fine.  If we just read dangling pointer or do undefined behavior, then yet, it should be 

The alternative is to take the Has*Handle  traits as parameter and keep ownership of it, but that also suck :-/

Reply:
Yep, all hell can break loose if the window is destroyed - AFAIK.